force-app/main/default/classes/ProgramPlanSettingsVMapper.cls

```diff
@@ -0,0 +1,77 @@
+    * Neither the name of Salesforce.org nor the names of
+    its contributors may be used to endorse or promote products derived
+    from this software without specific prior written permission.
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+ * @author Salesforce.org
+ * @date 2021
+ * @group EDA Settings
+ * @group-content ../../ApexDocContent/EdaSettings.htm
+ * @description View mapper to generate view models associated with Program Plan Settings
+ */
+public virtual with sharing class ProgramPlanSettingsVMapper {
+    /**********************************************************************************
+     * @description Instance for Singleton Pattern
+     **********************************************************************************/
+    @TestVisible
+    private static ProgramPlanSettingsVMapper instance;
+
+    /**********************************************************************************
+     * @description Empty constructor is not public for Singleton Pattern
+     **********************************************************************************/
+    public ProgramPlanSettingsVMapper() {
+    }
+
+    /**********************************************************************************
+     * @description Static method to get the current instance for Singleton pattern
+     * @return The instance of AffiliationsSettingsVMapper.
+     **********************************************************************************/
+    public static ProgramPlanSettingsVMapper getInstance() {
+        if (instance == null) {
+            instance = new ProgramPlanSettingsVMapper();
+        }
+
+        return instance;
+    }
+
+    /**********************************************************************************************************
+     * @description Retrieve an instance of ProgramPlanSettingsVModel
+     * @return An instance of ProgramPlanSettingsVModel
+     *********************************************************************************************************/
+    public virtual ProgramPlanSettingsVModel getProgramPlanSettingsVModel() {
+        ProgramPlanSettingsModel ppModel = locateProgramPlanSettingsService().getProgramPlanSettingsModel();
```

Comment:
Let's make the variable name more specific like programPlanSettingsModel. Abbreviation sometimes cause confusion. Plus the full variable name isn't that bad.

Reply:
I fixed this here and in other places